src/Tests/StateTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/**
+ * @file
+ * Definition of Drupal\acquia_search\Tests\AcquiaConnectorSearchTest.
```

Comment:
^^^^^^^^^^^^^^^^^^^^^^^^^, beware that also the class loader is not going to pick it up that way. The last part is actually parsed!!